### PR TITLE
Update node-ingest to support latest version

### PR DESCRIPTION
**Pull request by AsafMah**
Changes:
- Download specific version, so the guide will always work
- Used old-style imports for older node.js clients
- Fixed deprecated imports
- Used new ctor for IngestionProperties
- Prettier printing

### CODE REVIEW

**Comment by PRMerger-2** [comment]
@AsafMah : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/node-ingest-data.md [hunk 1]
```diff
@@ -66,7 +64,7 @@ const kustoIngestUri = `https://ingest-${cluster}.${region}.kusto.windows.net`;
 const kustoDatabase  = "Weather";
 ```
 
-Now construct the connection string. This example uses device authentication to access the cluster. You can also use Azure Active Directory application certificate, application key, and user and password.
+Now construct the connection string. This example uses device authentication to access the cluster (Make sure to look at the console output to authenticate). You can also use Azure Active Directory application certificate, application key, and user and password.
```
Comment (shsagir) [inline data-explorer/node-ingest-data.md:67]:
```suggestion
Now construct the connection string. This example uses device authentication to access the cluster. Check the console output to complete the authentication. You can also use an Azure Active Directory application certificate, application key, and user and password.
```

**Comment by PRMerger18** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**